packages/connection-form/src/components/connection-form.spec.tsx

```diff
@@ -243,7 +242,10 @@ describe('ConnectionForm Component', function () {
     );
   });
 
-  context('when preferences.showFavoriteActions === false', function () {
+  // TODO: with enableNewMultipleConnectionSystem: true
```

Comment:
some TODOs in this file

Reply:
For everything in the connection-form I'd maybe just link to the existing ticket to fix the form for VSCode, it's broken right now anyway and this ticket is a good opportunity to address these tests too